index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio Project</title>
+</head>
+<body>
+    <div class="body-container">
+      <header class="toolbar">
+        <a href="#" class="logo">John Doe</a>
+      <nav class="navbar-container">
+        <ul>
+          <li><a class = "hamburger" href="javascript:void(0);"><i class="fa fa-bars"></i></a></li>
+        </ul>
+      </nav>
+    </header>
+      <div>
+        <img src="images/ellipse18.png" class="ellipse-image-one" alt="ellipse-image-one"/>
+      </div>
+      <div class="section1-details">
+        <section>
+            <h1>Paul Onchera</h1>
+            <h3>Software Developer</h3>
+            <p>I can help you build a product , 
+               feature or website Look through 
+               some of my work and experience! If 
+               you like what you see and have a 
+               project you need coded,  don’t 
+               hestiate to contact me.
+            </p>
```

Comment:
## OPTIONAL

- [ ] Kindly add the list of frameworks, languages and skills and also list of social media icons to this section, as  this is part of this template's design 👍🏿 

![Template 4-main-mobile](https://user-images.githubusercontent.com/69638013/164461905-c9770fed-7375-4d17-8d32-0d86431f201c.png)